build/Targets/Settings.props

```diff
@@ -34,8 +34,9 @@
     <RoslynDiagnosticsPropsFilePath>$(NuGetPackageRoot)/microsoft.net.roslyndiagnostics/$(RoslynDiagnosticsNugetPackageVersion)/build/Microsoft.Net.RoslynDiagnostics.props</RoslynDiagnosticsPropsFilePath>
     <RoslynPortableTargetFrameworks>net461;netcoreapp2.0</RoslynPortableTargetFrameworks>
     <RoslynPortableTargetFrameworks46>net46;netcoreapp2.0</RoslynPortableTargetFrameworks46>
-    <RoslynPortableRuntimeIdentifiers>win7;win7-x64</RoslynPortableRuntimeIdentifiers>
- 
+    <RoslynPortableRuntimeIdentifiers>win7;win7-x64;linux-x64;osx-x64</RoslynPortableRuntimeIdentifiers>
```

Comment:
Should that be `win7-x86;win7-x64;...`? It'll probably work as long as we don't have any native dependencies but would be slightly more correct I think?

Reply:
(yes, I recognize it's the wrong thing already, but thought I'd ask...)

Reply:
I think this is another hold over from the project.json days. I wonder if we could just make it win7 flat? But I've never been super strong an RIDs. 

We could explore that in another PR though. This is def moving us forward as is.

Reply:
AFAIK win7 means win7-x86. /cc @weshaggard

Reply:
The `win7` RID (and any RID that doesn't have an architecture) means it works for all architectures (or is architecture-agnostic).  Check out https://github.com/dotnet/corefx/tree/master/pkg/Microsoft.NETCore.Platforms#runtime-ids for a really good explanation of RIDs.

BTW - `win7-x64` can be changed to just `win-x64`, since that is the "portable" RID for Windows.